.github/workflows/build.yml

```diff
@@ -75,6 +83,23 @@ jobs:
                     restore-keys: |
                         ${{ runner.os }}-php-${{ matrix.php }}-composer-
 
+            -
+                name: Install Sylius version
+                run: |
+                    composer require "sylius/sylius:${{ matrix.sylius }}" --no-update
```

Comment:
I'm not sure if this is necessary as we have `Restrict Sylius version` in line 99, could you check that?